witchcraft/internal/tcpjson/tcp_logger_test.go

```diff
@@ -188,15 +189,17 @@ func getEnvelopeBytes(t *testing.T, payload []byte) []byte {
 // bytes buffer instead of to the net.Conn.
 type bufferedConnProvider struct {
 	net.Conn
-	err    error
-	buffer bytes.Buffer
+	err        error
+	buffer     bytes.Buffer
+	writeCount int32
```

Comment:
Can you add a comment that this is only used in testing?

Reply:
This conn provider can only be used from `_test.go` files, but I added the comment anyways.

Reply:
oh whoops! I didn't realize this was in a test file. My bad